lib/browser/storage.js

```diff
@@ -22,17 +22,25 @@ try {
 
 export const canStoreURLs = hasStorage;
 
-export function setItem(key, value) {
-  if (!hasStorage) return;
-  return localStorage.setItem(key, value);
-}
+export class DefaultStorage {
```

Comment:
A more detailed name, such as `LocalStorage` or `WebStorage` (?), would be appropriate.